exchanges/binance/binance_websocket.go

```diff
@@ -454,66 +455,158 @@ func (b *Binance) UpdateLocalBuffer(wsdp *WebsocketDepthStream) error {
 		return err
 	}
 
-	currencyPair, err := currency.NewPairFromFormattedPairs(wsdp.Pair,
+	currencyPair, err := currency.NewPairFromFormattedPairs(u.Pair,
 		enabledPairs,
 		format)
 	if err != nil {
 		return err
 	}
 
-	currentBook := b.Websocket.Orderbook.GetOrderbook(currencyPair, asset.Spot)
+	return b.CheckAndLoadBook(currencyPair, asset.Spot, u)
```

Comment:
I understand that spot and margin orderbooks are the same according to Binance, but there are complications that arise from these assumptions.
- If you have both spot and margin enabled and you wish to stream the orderbook via the following command `.\gctcli.exe getorderbookstream --exchange=binance --pair=ltc-usdt --asset=spot` , only the spot version gets websocket updates
- If you only have margin enabled and run `.\gctcli.exe getorderbookstream --exchange=binance --pair=ltc-usdt --asset=margin`, you will receive `orderbook item not found for binance ltcusdt margin`, even though the orderbook exists and is getting updates.
  - Further to this, you also cannot retrieve via spot, since the asset is not enabled